examples/Util.re

```diff
@@ -0,0 +1,22 @@
+let wrapUnicodeString = (input: string) => {j|$input|j};
```

Comment:
I don't think this is needed/has any effect?

It compiles to

```re
function wrapUnicodeString(input) {
  return "" + (String(input) + "");
}
```

Reply:
I believe you're right, it's something stupid I did at the very beginning. Done.